os/arch/arm/src/armv8-m/up_mpu.c

```diff
@@ -187,3 +187,38 @@ void up_mpu_set_register(uint32_t *mpu_regs)
 		putreg32(mpu_regs[REG_RASR], MPU_RLAR);
 	}
 }
+
+/****************************************************************************
+ * Name: up_mpu_check_active
+ *
+ * Description:
+ *   Checks if mpu registers passed as argument are the ones set in
+ *   mpu h/w
+ *
+ ****************************************************************************/
+bool up_mpu_check_active(uint32_t *mpu_regs)
```

Comment:
Need to include <stdbool.h> for bool
include <up_arch.h> for putreg32
include <tinyara/mpu.h> for up_mpu_set_register

Reply:
same as v7-m

Reply:
Done.